CMakeLists.txt

```diff
@@ -1,3 +1,50 @@
+# ========================================
+# Linux
+# ========================================
+# $ cd /home/${USER}/git/xeve
+# $ mkdir build_main
+# $ cd build_main
+#
+# $ cmake ..
+# $ make -j <threads>
+# $ make install
+# $ make package
+
+# ========================================
+# Windows
+# ========================================
+#
+# ----------------------------------------
+# MingGW 64 with MSVC compiler
+# ----------------------------------------
+#
+# 1. Open "x64 Native Tools Command Prompt for VS 2019" and write the following command
+#
+# c:\msys64>msys2_shell.cmd -mingw64
+#
+# 2. MinGW shell
+#
+# $ cd /home/${USER}/git/xeve
+# $ mkdir build_main
+# $ cd build_main
+# $ cmake .. -G "MinGW Makefiles" -DCMAKE_C_COMPILER=cl -DCMAKE_CXX_COMPILER=cl -DMSVC=true -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DCMAKE_INSTALL_PREFIX=/usr
+# $ cmake --build . -j <threads>
+# $ cmake --install .
+# $ cmake --build . --package
+#
+# ----------------------------------------
+# MingGW 64 with gcc compiler
+# ----------------------------------------
+# MinGW shell
+#
+# $ cd /home/${USER}/git/xeve
+# $ mkdir build_main
+# $ cd build_main
+# $ cmake .. -G "MinGW Makefiles" -DCMAKE_INSTALL_PREFIX=/usr
+# $ cmake --build . -j <threads>
+# $ cmake --install .
+# $ cmake --build . --package
+
```

Comment:
In my opinion it will be better to place this guide on project wiki page, not here.